aspnetcore/signalr/streaming.md

```diff
@@ -52,7 +52,7 @@ Streaming hub methods can return `IAsyncEnumerable<T>` in addition to `ChannelRe
 
 ::: moniker-end
 
-The following sample shows the basics of streaming data to the client using Channels. Whenever an object is written to the <xref:System.Threading.Channels.ChannelWriter`1>, the object is immediately sent to the client. At the end, the `ChannelWriter` is completed to tell the client the stream is closed.
+The following sample shows the basics of streaming data to the client using Channels. Whenever an object is written to the <xref:System.Threading.Channels.ChannelWriter%601>, the object is immediately sent to the client. At the end, the `ChannelWriter` is completed to tell the client the stream is closed.
```

Comment:
IIRC this was done on purpose when the doc was written. I think the `%60` broke. I'll check the build tho. Maybe 2nd time is the charm! 🍀:smile:

Reply:
*Oh, cool!* It passed this time. Leave it ... we were going to do that anyway until we hit a snag on the original PR (either it was this one, or I might be thinking of another one where we tried this and it :boom: on us).